### DISCUSSION TITLE
Is there a way to allow drag n drop between 2 computers ?

### DISCUSSION THREAD

**Discussion by vEonz121** [input-leap/input-leap#1509] [Q&A] [answered]
Is there an auto upload-download of a file from source to destination just by dragging a file from one machine to the other using inputleap?

if not with this repo, is there any software that can be compatible with inputleap to achieve this ?

**Comment by shymega** [comment, marked as answer]
No, and I don't know of any other software that does that, let alone be compatible with IL.

**Comment by JTech2022** [comment]
Is there a possibility to use the cloud of telegram to allow this future somehow?

**Reply by shymega** [reply]
No.

**Reply by JTech2022** [reply]
The fact that this its working means that barrier does not?

**Reply by shymega** [reply]
I don't understand what you mean.
